crates/common/tedge_utils/src/file.rs

```diff
@@ -325,6 +325,7 @@ impl PermissionEntry {
                             from: e,
                         })
                         .await?;
+                    f.flush().await?;
```

Comment:
The current sequence of calls (`sync_all`, `write_all`, `flush`) has to be rewritten as  (`write_all`, `flush`, `sync_all`)

Reply:
note that [`flush` on a `File` is a noop](https://doc.rust-lang.org/std/fs/struct.File.html#platform-specific-behavior-14) so `sync_all` probably suffices, but we can keep the `flush` to be consistent.